language-plutus-core/src/Language/PlutusCore/Evaluation/Machine/Ck.hs

```diff
@@ -45,6 +45,7 @@ type CkM uni = Either (CkEvaluationException uni)
 
 instance SpendBudget (CkM uni) uni where
     spendBudget _ _ _ = pure ()
+    builtinCostParams = pure defaultCostingFunParameters
```

Comment:
Hm, I don't feel great about this. That said, the CK machine doesn't handle extensible builtins either. Probably we should refit it to at least work in an appropriate `Reader` over the builtin meanings and the cost paramaters. But that can be a TODO (but write the TODO in please).

Reply:
IIRC it doesn't do budgeting at all.